### ISSUE TITLE
Library 'mosekscopt8_1.dylib' not found 

### ISSUE THREAD

**Issue by MFairley** [JuliaOpt/Mosek.jl#134]
I am on Mac OS 10.13.1 with Julia 0.6.1 and getting the following build error:

LoadError: Library 'mosekscopt8_1.dylib' not found (/Users/mfairley/.julia/v0.6/Mosek/deps/src/mosek/8/tools/platform/osx64x86/bin)

I see the same issue also reported in #126 so I thought the issue was closed and fixed.

Running 
Pkg.checkout("Mosek","master")

gives a new build error

INFO: Building Mosek
INFO: Get latest MOSEK version (https://d2i6rjz61faulo.cloudfront.net/stable/9/version)
INFO: Latest MOSEK version = <?xml version="1.0" encoding="UTF-8"?>
<Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Key>stable/9/version</Key><RequestId>E99DC452D7824344</RequestId><HostId>ry25Qxlqn+AO/J/BKOHWMTSzOwGVrzgqxSKCFiBHJoyYnZgW0wo+BPwQvpfYzYimtwGAE3UanRU=</HostId></Error>, currently installed = 8.1.0.33
INFO: Download MOSEK distro (https://d2i6rjz61faulo.cloudfront.net/stable/<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Key>stable/9/version</Key><RequestId>E99DC452D7824344</RequestId><HostId>ry25Qxlqn+AO/J/BKOHWMTSzOwGVrzgqxSKCFiBHJoyYnZgW0wo+BPwQvpfYzYimtwGAE3UanRU=</HostId></Error>/mosektoolsosx64x86.tar.bz2)
=======================================================[ ERROR: Mosek ]========================================================

LoadError: Failed to download MOSEK distro
while loading /Users/mfairley/.julia/v0.6/Mosek/deps/build.jl, in expression starting on line 196

**Comment by chriscoey** [comment]
Hi Michael,
You can't be on master at the moment. Things are a little broken as Mosek 9 is not yet released, and Mosek.jl is in transition to MathOptInterface from MathProgBase.
Things should work if you don't try to checkout master.

**Comment by MFairley** [comment]
Hi Chris, the first error I mentioned is what I’m getting using the stable build from Pkg.add(“Mosek”). I tried to remedy the issue by switching to master but that resulted in the second error message.

**Comment by chriscoey** [comment]
Ah ok, hopefully @ulfworsoe will be able to help then.

**Comment by MFairley** [comment]
Changing line 33 of build.jl to the following fixed my issue:

`elseif is_apple()   "libmosek64.$(mskvmajor).$(mskvminor).dylib", "libmosekscopt$(mskvmajor)_$(mskvminor).dylib"`

**Comment by ulfworsoe** [comment]
Yes, master should be generally avoided at the moment, but I have created branches b0.6, b0.7 and b0.8 that contains revisions that are not released yet. You can try to do
```
Pkg.checkout("Mosek","b0.7")
Pkg.build("Mosek")
```
